README.md

```diff
@@ -146,76 +141,65 @@ To use this form, render a link on your page to:
 
 [[Back to the top]](#top)
 
-## Report an Accessibility Problem <a name="report-an-accessibility-problem"></a>
+### Contacting HMRC - *Help and contact*
+> :warning: This form is deprecated, as experience has shown that it is used to report things that DTST can't help with,
```

Comment:
added deprecation warning and removed technical details as we don't want new teams using this